src/set_theory/surreal.lean

```diff
@@ -198,6 +202,166 @@ theorem numeric_nat : Π (n : ℕ), numeric n
 theorem numeric_omega : numeric omega :=
 ⟨by rintros ⟨⟩ ⟨⟩, λ i, numeric_nat i.down, by rintros ⟨⟩⟩
 
+theorem zero_lt_one : (0 : pgame) < 1 :=
+begin
+  rw lt_def,
+  left,
+  use ⟨punit.star, by split; rintro ⟨ ⟩⟩,
+end
```

Comment:
This, and much of the following content about `half`, should presumably be in `pgame.lean`.

Reply:
I'll move the code.

My thinking was that these theorems are probably not going to be used for anything other than constructing the dyadic surreals.